ChangeLog.md

```diff
@@ -20,6 +20,12 @@ See docs/process.md for more on how version tagging works.
 
 3.1.32 (in development)
 -----------------------
+- In Wasm exception mode (`-fwasm-exceptions`), when
+  `EXCEPTION_STACK_TRACES` is enabled, uncaught exceptions will display stack
+  traces. This defaults to true when `ASSERTIONS` is enabled. This option is
```

Comment:
Hmm, I think this should be "This is set to true when `ASSERTIONS` is enabled", because we print exception stack traces even when `-sASSERTIONS -sEXCEPTION_STACK_TRACES=0`. See https://github.com/emscripten-core/emscripten/pull/18642#discussion_r1093675794

Reply:
If we ban `ASSERTIONS=0 && EXCEPTION_TRACES=1` as suggested in https://github.com/emscripten-core/emscripten/pull/18642#discussion_r1093682938, I think the current text is OK.